src/datachain/__init__.py

```diff
@@ -74,6 +76,8 @@
     "create_project",
     "datasets",
     "delete_dataset",
+    "delete_namespace",
+    "delete_project",
```

Comment:
Let's don't expose project in user's api. Instead you can do something like:

```python
def delete_namespace(name: str):
    parts = name.split(".")
    if len(parts) == 1:
        remove_namespace(parts[0])
    elif len(parts) == 2:
        delete_project(parts[1])
    else:
        raise ValueError(f"Invalid namespace format: {name}. Expected 'namespace' or 'ns1.ns2'.")
```

Reply:
Not sure I follow 🙂

We’re exposing an API that’ll be removed soon anyway. Skipping it would just mean ~5 extra lines of code.

Or am I missing something?